ui/selectors/accounts.ts

```diff
@@ -35,6 +35,11 @@ export const getInternalAccounts = createSelector(
   (accounts) => accounts,
 );
 
+export const getInternalAccountsObject = createSelector(
```

Comment:
we should add a simple test to this selector

Reply:
Done: [4af02f2](https://github.com/MetaMask/metamask-extension/pull/34884/commits/4af02f225f7775f4e0ef8e04adcc690b2bd81e0b)

Reply:
Done: [b67cf91](https://github.com/MetaMask/metamask-extension/pull/34884/commits/b67cf9119f45fd9700b89319f6757fe3d1caf5c0)